src/components/language/language.js

```diff
@@ -76,6 +79,38 @@ class Language extends Component {
    */
   componentDidMount() {
     this.props.eventManager.listen(document, 'click', e => this.handleClickOutside(e));
+    this.props.addKeyboardHandler({code: KeyMap.C}, event => {
+      event.preventDefault();
```

Comment:
import this handler from `keyboard.js`. we can't maintain code like this twice.
same for all new handlers in this change

Reply:
we'll not.
`keyboard.js` is deprecated if someone wants to use it he'll need to write the logic by himself I don't think we need to maintain this code anymore.